### PR TITLE
[azsystemevents] Updating to release commit for April ca560e400087f54b306f3a21df582c8a5bbd0abc

### PR DESCRIPTION
**Pull request by richardpark-msft**
* Renames that came out of the arch board.
* 'string' became a fleshed out enumeration

### CODE REVIEW

File: sdk/messaging/eventgrid/azsystemevents/constants.go [hunk 1]
```diff
@@ -148,7 +172,6 @@ const (
 	ACSRouterUpdatedWorkerPropertyAvailableForOffers    ACSRouterUpdatedWorkerProperty = "AvailableForOffers"
 	ACSRouterUpdatedWorkerPropertyChannelConfigurations ACSRouterUpdatedWorkerProperty = "ChannelConfigurations"
 	ACSRouterUpdatedWorkerPropertyLabels                ACSRouterUpdatedWorkerProperty = "Labels"
-	ACSRouterUpdatedWorkerPropertyMaxConcurrentOffers   ACSRouterUpdatedWorkerProperty = "MaxConcurrentOffers"
```
Comment (chlowell) [inline sdk/messaging/eventgrid/azsystemevents/constants.go:151]: Should this, ChannelType -> ChannelKind, and the model field renames be in the changelog?

**Review by chlowell — APPROVED** [review]

File: sdk/messaging/eventgrid/azsystemevents/constants.go (hunk 1 above)
Comment (richardpark-msft) [reply sdk/messaging/eventgrid/azsystemevents/constants.go:151]: Well, "should" was NOT true :) I've updated the changelog to incorporate all the renames. It wasn't too many. I also added in a rename for the RecordingChannelType (which was the only one I saw that hadn't gotten the rename treatment).

**Pull request merged**